go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/envoyproxy/go-control-plane v0.9.8-0.20201019204000-12785f608982
 	github.com/evanphx/json-patch v4.9.0+incompatible
+	github.com/evanphx/json-patch/v5 v5.1.0
```

Comment:
do we need v5 or does v4 fulfill the same requirements?

Reply:
v4 should work, v5 has some bugfixes and will be actively maintained though.